azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -69,6 +69,19 @@
 CPUQuota={0}%
 """
 _AGENT_CPU_QUOTA = 5
+_AGENT_THROTTLED_TIME_THRESHOLD = 120  # 2 minutes
+
+
```

Comment:
The CGroupsConfigurator instance is shared by the extension handler and monitor threads (monitor invokes the checks on cgroups). As I add more logic to those checks, I need to be more strict with the internal state of the configurator. I made several methods static.

These 2 loggin methods used to be instance methods; making them static would require prefixing each invocation with "CGroupsConfigurator._Impl", which hurts readability so I made them global (but internal) to the module.